VsViewer/vs_calc/CPT.py

```diff
@@ -170,14 +259,32 @@ def from_json(json: Dict):
         )
 
     @staticmethod
-    def from_file(cpt_ffp: str):
+    def from_file(cpt_ffp: str, gwl: float = 1, nar: float = 0.8):
         """
```

Comment:
Not really a big deal here, but constructors like this really ought to be [class methods](https://docs.python.org/3/library/functions.html#classmethod) rather than static methods as in Java because they behave better with subclassing.